tests/application-gateway-legacy-tests/README.md

```diff
@@ -0,0 +1,26 @@
+# Application Gateway Tests
+
+## Overview
+
+This project contains the acceptance tests for the Kyma Application Gateway.
+
+## Prerequisites
+
+The project requires Go 1.8 or higher.
+
+## Usage
+
+Environment parameters used by the tests:
+
+| Name | Required | Default | Description | Possible values |
```

Comment:
See the [guidelines for tables](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md#tables).
Although we have some inconsistencies in our documentation, our tables generally follow the order: 
```
| Parameter      | Required | Description   | Default value |       {PROFILE} value       | Example value   |
|----------------|:--------:|---------------|---------------|-----------------------------|-----------------|
| {Param's name} | {Yes/No} | {Description} |      None     | {Value for the given prof.} | {Example value} |
```
For consistency's sake, we should stick to the same order when creating tables.

Reply:
At first I will switch Default and Description columns. I will lookup for PROFILE values then.

Reply:
You don't need to apply all the columns. Choose the ones you need, just stick to the order. The profile value refers to tables like [this one](https://sap.sharepoint.com/sites/100093/_layouts/15/Doc.aspx?sourcedoc={d9b81bfd-650d-4e42-9f62-43220886f783}&action=edit&wd=target%28SIG%20Content.one%7C0a2f8356-73ae-4f4a-adf6-0a3617f99e51%2FRelease%20notes%201.16%20%28input%5C%29%7C22d7a1e1-38b5-449d-8b8d-b42c3745632c%2F%29).